### PR TITLE
fix(Webjerk): Updated markdown file example titles

### PR DESCRIPTION
**Pull request by andrewleyva**
Webjerk uses titles of examples to create the filenames of the images it uses for diffs. If an example does not have a title, the filename is created with the dom index of the example. The problem is this dom index can change when you add examples.
To resolve this, I have updated examples with a title and then ran webjerk to reset the refs correctly.

# problem statement

Not all examples were formatted properly with a title. Webjerk uses titles of examples to create the filenames of the images it uses for diffs. If an example does not have a title, the filename is created with the dom index of the example. The problem is this DOM index can change when you add examples.

Basically, tests may fail when the components are fine.

# solution

To resolve this, I have updated examples with a title and then ran webjerk to reset the refs correctly.

### CODE REVIEW

**Comment by cdaringe** [comment]
woo hoo! 🎉  +1. thx
